pkg/services/action_test.go

```diff
@@ -304,63 +299,6 @@ func (s *ServicesTestSuite) TestNewSubmitBizCase() {
 		s.Equal(1, sendReviewerEmailCount)
 	})
 
-	s.Run("submit Biz Case sets the intake status to the value passed", func() {
```

Comment:
double check - was this ok to remove?

Reply:
test removed as it was focused around setting an intake status to a passed value, but we are removing value, so I removed the test